packages/sku/lib/program/options/site/site.option.js

```diff
@@ -0,0 +1,4 @@
+const { Option } = require('commander');
+const siteOption = new Option('-si, --site [site]', 'Site to serve on');
```

Comment:
Honestly not a fan of `-si`, and I think this sort of breaks short-hand flag composition, e.g. if `-s` and `-i` were both boolean flags you could do `-si` to set both.

Even though those aren't boolean flags in sku, I'd rather not confuse anyone that expects `-si` to set two flags. Since `-s` is already taken by `--stats`, I'd just remove the shorthand for this option.

Reply:
Good call. This should just be a single flag without a shortcut. I think I mentioned it without a shortcut in the PR description.